examples/webgl2_msaa.html

```diff
@@ -0,0 +1,174 @@
+				scene.background = new THREE.Color( 0xffffff );
+				scene.fog = new THREE.Fog( 0xcccccc, 100, 1500 );
+
+				clock = new THREE.Clock();
+
+				//
+
+				const hemiLight = new THREE.HemisphereLight( 0xffffff, 0x222222, 1.5 );
+				hemiLight.position.set( 1, 1, 1 );
+				scene.add( hemiLight );
+
+				//
+
+				group = new THREE.Group();
+
+				const geometry = new THREE.SphereGeometry( 10, 64, 40 );
+				const material = new THREE.MeshLambertMaterial( { color: 0xee0808 } );
+				const material2 = new THREE.MeshBasicMaterial( { color: 0xffffff, wireframe: true } );
+
+				for ( let i = 0; i < 10; i ++ ) {
+
+					const mesh = new THREE.Mesh( geometry, material );
+					mesh.position.x = Math.random() * 600 - 300;
+					mesh.position.y = Math.random() * 600 - 300;
+					mesh.position.z = Math.random() * 600 - 300;
+					mesh.rotation.x = Math.random();
+					mesh.rotation.z = Math.random();
+					mesh.scale.setScalar( Math.random() * 5 + 5 );
+					group.add( mesh );
+
+					const mesh2 = new THREE.Mesh( geometry, material2 );
+					mesh2.position.copy( mesh.position );
+					mesh2.rotation.copy( mesh.rotation );
+					mesh2.scale.copy( mesh.scale );
+					group.add( mesh2 );
+
+				}
+
+				scene.add( group );
+
+				//
+
+				renderer = new THREE.WebGLRenderer();
+				renderer.setPixelRatio( window.devicePixelRatio );
+				renderer.setSize( window.innerWidth, window.innerHeight );
+				document.body.appendChild( renderer.domElement );
+
+				//
+
+				const parameters = {
+					format: THREE.RGBFormat
+				};
+
+				renderTarget = createRenderTarget( parameters );
+
+				//
+
+				window.addEventListener( 'resize', onWindowResize );
+
+				const gui = new GUI();
+				gui.add( params, 'samples', [ 0, 4, 8, 16 ] ).onChange( ( value ) => {
+
+					renderTarget.dispose();
+					renderTarget = createRenderTarget( parameters, parseFloat( value ) );
+
+				} );
+				gui.add( params, 'autoRotate' );
+				gui.open();
+
+				animate();
+
+			}
+
+			function createRenderTarget( parameters, samples = 4 ) {
+
+				const size = renderer.getDrawingBufferSize( new THREE.Vector2() );
+
+				const renderTarget = new THREE.WebGLMultisampleRenderTarget( size.width, size.height, parameters );
+				renderTarget.samples = samples;
+				renderTarget.toScreen = true; // this will resolve the multisampled render target to the default framebuffer (screen)
+
+				return renderTarget;
+
+			}
+
+			function onWindowResize() {
+
+				camera.aspect = window.innerWidth / window.innerHeight;
+				camera.updateProjectionMatrix();
+
+				renderer.setSize( window.innerWidth, window.innerHeight );
+
+				const size = renderer.getDrawingBufferSize( new THREE.Vector2() );
+				renderTarget.setSize( size.width, size.height );
+
+			}
+
+			function animate() {
+
+				requestAnimationFrame( animate );
+
+				if ( params.autoRotate ) group.rotation.y += clock.getDelta() * 0.1;
+
+				renderer.setRenderTarget( renderTarget );
+				renderer.render( scene, camera );
```

Comment:
This API usage is actually a bit unusual since applying a render target via `setRenderTarget()` will still render to screen (and not into the render target). 

I though about applying the render target as a context parameter when creating the renderer. However, this means an enhancement of `WebGLRenderer` will be required since it has to manage an additional render target and apply it properly.